src/stylesheets/blocks/_key-factors.scss

```diff
@@ -14,17 +14,30 @@
   display: block;
   margin: auto;
 
+  @media (screen(lg)) {
+    height: rem(800);
+    width: rem(1000);
+    background-size: rem(1000);
+    margin: rem(-40) auto auto auto;
+  }
```

Comment:
It looks like the key-factors section and the behind-the-scenes sections share some styles. Can you think of a way to share those styles cleanly?